### PR TITLE
Bug fix 3.5/traversal cache

### PR DESCRIPTION
**Pull request by hkernbach**
Same as: https://github.com/arangodb/arangodb/pull/10854

### CODE REVIEW

**Comment by hkernbach** [comment]
http://jenkins01:8080/view/PR/job/arangodb-matrix-pr/7965/

**Comment by hkernbach** [comment]
http://jenkins01:8080/view/PR/job/arangodb-matrix-pr/7995/

**Comment by hkernbach** [comment]
http://jenkins01:8080/view/PR/job/arangodb-matrix-pr/8027/

**Comment by hkernbach** [comment]
http://jenkins01:8080/job/arangodb-matrix-pr-linux/9811/
Jenkins green.

File: arangod/Cluster/ClusterMethods.cpp [hunk 1]
```diff
@@ -2043,117 +2042,13 @@ int fetchEdgesFromEngines(std::string const& dbname,
 ///        If no server responds with a document
 ///        a 'null' will be inserted into the result.
 
-void fetchVerticesFromEngines(
```
Comment (mchacki) [inline arangod/Cluster/ClusterMethods.cpp]: I much like this :+1:

**Pull request merged**